src/io.jl

```diff
@@ -170,7 +168,7 @@ function get_gen_data(
         "Time from Cold Shutdown to Full Load",
         "Status"
     ]
-    df = CSV.read(datapath, rows_for_type_detect=typemax(Int))
```

Comment:
IIRC, the `rows_for_type_detect=typemax(Int)` used to be necessary in 0.6 for it to determine types that were general enough. Can you check if it is not choosing types that are too constrained without it?

Reply:
As of `v0.5` for CSV `rows_for_type_detect` is gone and no longer needed.

> https://discourse.julialang.org/t/ann-new-csv-jl-0-5-release/23848